src/NodeMetric.tsx

```diff
@@ -57,7 +58,13 @@ export const NodeMetric = ({ width, height, data, timeRange, setShowGraph, focus
             />
           </div>
         </div>
-        <div className="NodeGraph">
+        <div
+          className={css`
+            width: 100%;
+            display: flex;
+            justify-content: center;
+          `}
+        >
```

Comment:
couldn't this one be also made with the styles class like the other (zb. styles.nodeGraph)? or do you think it's better with emotion?